.evergreen/config.yml

```diff
@@ -1017,8 +1017,8 @@ buildvariants:
     distros:
     - ubuntu2004-small
 - name: macos_x86_64
-  display_name: "macOS (x86_64) 10.14"
-  run_on: macos-1014
+  display_name: "macOS (x86_64) 14"
+  run_on: macos-14
```

Comment:
```suggestion
  display_name: "macOS (x86_64) 11"
  run_on: macos-11
```

Reply:
Whoops, forgot to add my local changes after patch testing. Fixed now!